assemblies/rosa-manage-objects-cli.adoc

```diff
@@ -0,0 +1,20 @@
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-managing-objects-cli"]
+
+= Managing objects with `rosa`
+
+:context: rosa-managing-objects-cli
+
+[role="_abstract"]
+Managing objects with the `rosa` cli, such as adding admin users, managing clusters, and scheduling cluster upgrades.
```

Comment:
General note that these [role="_abstract"] items seem to be sentence fragments. I'm not sure how this renders in the output or what the convention is since we don't use that element, so if that is written as intended please disregard.

Reply:
In the assemblies, these fragments are used just for Google search. Character number is limited. Also, we are waiting on standards for these. I have actively proposed standards that help users intuitively know whether the search results is a procedure, concept, or reference topic. We'll see what the committee decides on, but we really don;t have to worry about this right now. No matter what, we'll end up retrofitting.